src/tokens/typography.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Typography tokens - Graphite Design System
+---
+
+# Typography tokens
+
+[[toc]]
+
+<p class="intro">Typography tokens are used to maintain a consistent set of font styles throughout your app.</p>
+
+## Font family
+
+| Token              | Value                                                                                                 | Example                                                                    |
+| :----------------- | :---------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `--gr-font-family` | -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif | <div class="font-sans">The quick brown fox jumped over the lazy dog.</div> |
```

Comment:
The original text contains `jumps` instead of `jumped`. Now you're missing the letter `s`

Reply:
Sharp, thanks. Will fix this.